src/ansys/fluent/core/fluent_connection.py

```diff
@@ -453,18 +454,20 @@ def __init__(
             self._connection_interface.get_cortex_connection_properties()
         )
         self._cleanup_on_exit = cleanup_on_exit
-
+        self._container = container
+        self._compose = os.getenv("PYFLUENT_USE_DOCKER_COMPOSE") or os.getenv(
```

Comment:
Instead of checking these env vars at vaious places, can we move these checks to method within launcher_utils.py? Note that, we generally compare boolean-type env-var values against "1" in pyfluent code.

Reply:
Yes, we can try this.

Reply:
Done. Thank you.

Reply:
@mkundu1 

When we have moved this to a single file and used the following cases, it is failing consistently.

1. compose = os.getenv("PYFLUENT_USE_DOCKER_COMPOSE") == "1" or os.getenv("PYFLUENT_USE_DOCKER_COMPOSE") == "1"

2. compose = os.getenv("PYFLUENT_USE_DOCKER_COMPOSE") or os.getenv("PYFLUENT_USE_DOCKER_COMPOSE")

Reply:
I have investigated it further, we are not importing the env var like this anywhere in the source code. We are evaluating it as per the requirement at each place. As per our initial discussion we were agreed that it will be OK to have evaluation of this env vars at various places. Reverting back to that change as test is failing consistently.

Reply:
Done. Thank you.